apps/nextjs-website/react-components/components/MegaHeader/MegaHeader.tsx

```diff
@@ -59,8 +59,26 @@ const MegaHeader = (props: MegaHeaderProps) => {
     };
   }, []);
 
+  useEffect(() => {
+    if (mobileMenuOpen) {
+      document.body.style.overflow = 'hidden';
+    } else {
+      document.body.style.overflow = '';
```

Comment:
```suggestion
      document.body.style.overflow = 'auto';
      document.body.style.touchAction = 'auto';
```
Explicitely set default value.
Follow Safari fix suggestion.